src/probnum/testproblems/__init__.py

```diff
@@ -0,0 +1 @@
+"""Test problems for probabilistic linear solvers."""
```

Comment:
is there a reason that the testproblems init has only this specific sentence here? Why only for the linear solver? :)

Reply:
Oops, that was meant to be somewhere else. Changed it to a general description.